charts/aws-pca-issuer/templates/hpa.yaml

```diff
@@ -7,6 +7,7 @@ apiVersion: autoscaling/v2beta1
 kind: HorizontalPodAutoscaler
 metadata:
   name: {{ include "aws-privateca-issuer.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I don't fully get what the behavior is when we don't have this fix in place - help me understand?

Reply:
Let me grab some logs of the before and after to validate, probably a good thing to confirm that it wasn't an issue with the fork.

Reply:
Thanks for explaining!